amundsen_common/models/table.py

```diff
@@ -142,3 +142,19 @@ class TableSchema(AttrsSchema):
     class Meta:
         target = Table
         register_as_scheme = True
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class TableSummary:
```

Comment:
should we put it in the faq on why we deprecate the popular table and add this one?

Reply:
> should we put it in the faq on why we deprecate the popular table and add this one?

Mmm... It's actually because of the name `popular_table` is not generic but changing name would break backward compatibility. This is internal structure, but do you think it's worth adding in FAQ? (https://github.com/lyft/amundsen/blob/master/docs/faq.md)